test-suite/issuers/issuers-advanced.spec.js

```diff
@@ -0,0 +1,28 @@
+
+const fixtures = require('../__fixtures__');
+
+describe('issuerAssertionMethods', ()=>{
+    fixtures.issuers.forEach((issuer)=>{
+        describe(issuer, ()=>{
+            fixtures.issuerAssertionMethods[issuer].forEach((assertionMethod)=>{
+                describe(assertionMethod, ()=>{
+                    fixtures.credentials.forEach((credential)=>{
+                        it('should issue ' + credential.type[1], async ()=>{
+                            let body = {
+                                credential, 
+                                options: {
+                                    proofPurpose: 'assertionMethod',
+                                    issuer: assertionMethod.split('#')[0],
```

Comment:
This is okay for now. There may be an edge case where one can't extract the `issuer` via splitting on the `#` character.

Reply:
yes, in such cases we would need to provide both... the main point here, is that we need a way to say who the issuer is AND a way to say which key to use... the former works with the current APIs, the latter is blocked by our lack of progress on support for "options" in the issuer api.